Js/main.js

```diff
@@ -11,11 +17,31 @@ function toggleMenue() {
     cross.classList.add('hidden');
   }
 }
-
-hamburger.addEventListener('click', () => toggleMenue());
-cross.addEventListener('click', () => toggleMenue());
-
-document.querySelectorAll('.menue-items').forEach((n) => n.addEventListener('click', () => {
-  menue.classList.add('hidden');
-  cross.classList.add('hidden');
-}));
\ No newline at end of file
+const togglePopup = () => {
+  if (popshow.classList.contains('hidden')) {
+    popshow.classList.remove('hidden');
+  } else {
+    popshow.classList.add('hidden');
+  }
+};
+hamburger.addEventListener('click', toggleMenue);
+cross.addEventListener('click', toggleMenue);
+document.querySelectorAll('.menue-items').forEach((n) => {
+  n.addEventListener('click', () => {
+    menue.classList.add('hidden');
+    cross.classList.add('hidden');
+  });
+});
+submit.forEach((button) => {
+  button.addEventListener('click', togglePopup);
+});
+hidePopup.addEventListener('click', togglePopup);
+form.addEventListener('submit', (e) => {
+  if (email.value.toLowerCase() !== email.value) {
+    e.preventDefault();
+    errorMsg.style.display = 'block';
+    errorMsg.innerHTML = 'Email must be in Lowercase';
```

Comment:
- [ ] Good job on showing the message, but the problem is that the message is getting shown at top of the contact section. even on my first tries, I got confused and could not see the message shown. so this is not a good user experience. kindly consider showing this message around the input fields or around submit button. you dont need to add the background color, and margin, paddings in this div. you can simply show a text message in red color if not having extra space around input or submit is a issue.